bets/__init__.py

```diff
@@ -104,6 +104,46 @@ def get_active_bets(self, project_id=None):
 
         return bets
 
+    def get_last_bets(self, project_id=None,
+                      states=None, types=None, due_last_hours=24):
+        '''Returns bets that expired or will expire not early N hours ago'''
```

Comment:
Re-formulate this somehow, please?

"Returns bets that are expired or will expire within last N hours", perhaps